src/pages/home.jsx

```diff
@@ -64,31 +66,33 @@ const Home = (props) => {
             
             const indexShuffleOption = Math.floor(Math.random() * shuffleOption.length);
             const indexRandomCell = shuffleOption[indexShuffleOption];
-  
             array[indexNull] = array[indexNull + indexRandomCell];
             array[indexNull + indexRandomCell] = null;
         }
   
         return array;
     }
   
-    const getCombination = () => {
-        let initialShuffleCount = getCountShuffle();
-        let mixedCombination = mixingArray(initialShuffleCount);
-        props.onStateCombination(mixedCombination);
+    const getCombinationForNewGame = () => {
```

Comment:
It is more about *setting* of combination, not *getting*. When you get something, you have a result. Here is a mutation, not a getting of something. Use `set` or `update`.